### PR TITLE
Report StateStatus in all cases except piped process is terminated

### LINKED ISSUE #1413: State applied status does not reflect the view in case deployment failed caused by some reasons other than piped process got killed
**What happened**:

In case deployment applied but return failed status, the view could not reflect the changed state correctly.

ref: https://github.com/pipe-cd/pipe/blob/98977685aa688cef282e92d4958b956c6a43ce3b/pkg/app/piped/controller/scheduler.go#L502-L504

**What you expected to happen**:

**How to reproduce it**:

**Environment**:
- `piped` version:
- `control-plane` version:
- Others:

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1413 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/executor/stopsignal.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (s *stopSignal) Signal() StopSignalType {
 	return StopSignalType(value)
 }
 
-func (s *stopSignal) Stopped() bool {
+func (s *stopSignal) Terminated() bool {
 	value := s.signal.Load()
-	return StopSignalType(value) != StopSignalNone
+	return StopSignalType(value) != StopSignalTerminate
```
Comment (nghialv) [inline pkg/app/piped/executor/stopsignal.go:106]: `==`

Comment (khanhtc1202) [reply pkg/app/piped/executor/stopsignal.go]: thanks 🙏

**Comment by nghialv** [comment]
Nice catch! Thank you.
/approve

**Pull request merged**